modules/openy_features/openy_node/modules/openy_node_alert/config/install/core.entity_form_display.node.alert.default.yml

```diff
@@ -179,6 +188,7 @@ content:
       match_operator: CONTAINS
       size: 60
       placeholder: ''
+      match_limit: 10
```

Comment:
what happens is more than 10?

Reply:
maybe we should keep it unlimited for not causing issues when 100+ nodes with same title

Reply:
Fixed